keps/sig-cloud-provider/0013-removing-in-tree-providers.md

```diff
@@ -0,0 +1,272 @@
+|-----------------------------------|------------------------------|----------------|------------------|
+| pkg/api/service                   |                              |                |                  |
+| pkg/api/v1/service                |                              |                |                  |
+| pkg/apis/core/v1/helper           |                              |                |                  |
+| pkg/controller                    |                              |                |                  |
+| pkg/credentialprovider/aws        |                              |                |                  |
+| pkg/master/ports                  |                              |                |                  |
+| pkg/cloudprovider                 | k8s.io/cloud-provider        | No             | Full             |
+| pkg/features                      |                              |                |                  |
+| pkg/kubelet/apis                  |                              |                |                  |
+| pkg/util/file                     | k8s.io/utils/file            | No             | Full             |
+| pkg/util/keymutex                 | k8s.io/utils/keymutex        | No             | Full             |
+| pkg/util/io                       | k8s.io/utils/io              | No             | Full             |
+| pkg/util/mount                    |                              |                |                  |
+| pkg/util/net/sets                 | k8s.io/utils/netsets         | No             | Full             |
+| pkg/util/node                     | k8s.io/cloud-provider/node   | No             | Partial          |
+| pkg/util/nsenter                  | k8s.io/utils/nsenter         |                |                  |
+| pkg/util/parsers                  |                              |                |                  |
+| pkg/util/strings                  | k8s.io/utils/strings         | No             | Full             |
+| pkg/version                       |                              |                |                  |
+| pkg/volume                        | k8s.io/cloud-provider/volume | Yes            | Partial          |
+| pkg/volume/util                   | k8s.io/cloud-provider/volume | Yes            | Partial          |
+| pkg/volume/util/volumepathhandler | k8s.io/cloud-provider/volume | Yes            | Partial          |
+
+#### Phase 2 - Building CCM from Provider Repos
+
+In Phase 2, providers will be expected to build cloud controller manager from their respective provider repos (`kubernetes/cloud-provider-<provider>`). Provider repos will be frequently synced from their respective staging repos by the kubernetes publishing bot. Development of providers are still done in `kubernetes/kubernetes/staging/cloudprovider/providers`.
+
+#### Phase 3 - Migrating Provider Code to Provider Repos
```

Comment:
Important to note here that we are not proposing that `k8s.io/cloud-provider-<provider>` will be vendored back into `k8s.iokubernetes`